### PR TITLE
Added to the ReadMe file

### PR DESCRIPTION
**Pull request by cskinner35**
Outlined sections, and wrote short messages under each.

### CODE REVIEW

**Review by pedrozaj — APPROVED** [review]
Test comment. Sections look good though. Leaving on to-do list due to additional needed sections.

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+## Introduction
+This is the introduction for the project.
+
+### Purpose
+There are two purposes of this project:
+1. To document examples of pull requests.
+2. That's it.
+
+### Scope
```
Comment (cskinner35) [inline README.md]: Commenting on a specific line of code.

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+Beyond providing an example of good documentation practices, there is no other usage for this specific repository. 
+However, the code contained herein is an excellent demonstration of a Dotnet project, that can be downloaded by the public. It showcases some of the features and power within the C# programming language and can be consulted for a great idea on how to develop for Windows desktop and devices.
```
Comment (pedrozaj) [inline README.md:13]: Example comment for a specific line of code.

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
...
+
+## Git the code
+Using a combination of GitHub and SourceTree, contributing to and tracking changes for this project is easier than ever!
+1. Make an account on GitHub.com, and contact the repository owner for an invite to be a Contributor.
+2. Use SourceTree to clone this repository to your local machine. The .git link can be found at the top of the repository's home page on GitHub.
+3. Use your favourite IDE to make changes to the code.
```
Comment (pedrozaj) [inline README.md:19]: Why did you use the British spelling of favorite?

**Review by pedrozaj — COMMENTED** [review]
Quick notes

File: README.md (hunk 3 above)
Comment (cskinner35) [reply README.md:19]: It's an alternative spelling.

Comment (pedrozaj) [reply README.md:19]: Okay, that's fine. However, in line 18, please change the text from 
'The .git link' to 'Our repository's  web URL'
Thanks.

Comment (cskinner35) [reply README.md:19]: Will do.

**Pull request closed without merging**